server/src/main/java/org/cloudfoundry/identity/uaa/client/VmidentityQueryableClientDetailsService.java

```diff
@@ -34,8 +34,7 @@ protected VmidentityQueryableClientDetailsService(MultitenantVmidentityClientDet
 
     @Override
     public List<ClientDetails> query(String filter, String sortBy, boolean ascending) {
```

Comment:
q: did you scan for all the filter in the code? i.e. did we properly fix up VmidentityApprovalStore ?